runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -444,6 +444,16 @@ class INTERPRETER_CLASS
 		return bp;
 	}
 
+	VMINLINE UDATA*
+	buildMethodFrameForDefaultConflictForMethodHandle(REGISTER_ARGS_LIST, J9Method *method, UDATA flags)
+	{
+		UDATA *bp = buildSpecialStackFrame(REGISTER_ARGS, J9SF_FRAME_TYPE_METHOD, flags, false);
+		*--_sp = (UDATA)method;
+		// default conflict method has no args
+		_arg0EA = bp;
+		return bp;
+	}
```

Comment:
this is only used once inside `throwDefaultConflictForMemberName` is there any reason to make this an inline helper?